cpu/cc2538/periph/flashpage.c

```diff
@@ -110,6 +119,6 @@ void flashpage_write(int page, const void *data)
 
     /* write page */
     if (data != NULL) {
-        flashpage_write_raw(page_addr, data, FLASHPAGE_SIZE);
+        flashpage_write(page_addr, data, FLASHPAGE_SIZE);
     }
 }
```

Comment:
We can now have a generic `flashpage_write_page()` implementation in `drivers/periph_common/flashpage.c`

```C
void flashpage_write_page(unsigned page, const void *data)
{
    assert((unsigned) page < FLASHPAGE_NUMOF);

    flashpage_erase(page);

    /* write page */
    if (data != NULL) {
        flashpage_write(flashpage_addr(page), data, FLASHPAGE_SIZE);
    }
}
```

Reply:
Good idea. I can add this here (or in a follow up) and guard it to only include it when `FLASHPAGE_SIZE` is available (it is not available on the stm32f{2,4,7}) and allow CPU implementations to override it if they have an optimized version available.

Reply:
Done and deduplicated